packages/react-components/react-text/package.json

```
@@ -33,6 +33,7 @@
   },
   "dependencies": {
     "@griffel/react": "^1.5.2",
+    "@fluentui/react-shared-contexts": "^9.2.0",
     "@fluentui/react-theme": "^9.1.5",
     "@fluentui/react-utilities": "^9.6.0",
```

_nit:_ Nice chance to order these alphabetically

```suggestion
    "@fluentui/react-shared-contexts": "^9.2.0",
    "@fluentui/react-theme": "^9.1.5",
    "@fluentui/react-utilities": "^9.6.0",
    "@griffel/react": "^1.5.2",
```